kratos/includes/model_part.h

```diff
@@ -515,6 +515,11 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
                 << "\" to the model part with name \"" << this->Name() << "\" which is not empty" << std::endl;
 
             mpVariablesList->Add(ThisVariable);
+
```

Comment:
Actually this should not be done
It even throws if nodes are present, see the lines above
We had a lot of memory problems, see #492